tests/test_default_qubit.py

```diff
@@ -376,7 +383,23 @@ class TestApply:
         ("T", [1 / math.sqrt(2), 1 / math.sqrt(2)], [1 / math.sqrt(2), np.exp(1j * np.pi / 4) / math.sqrt(2)]),
         ("Hadamard", [1, 0], [1/math.sqrt(2), 1/math.sqrt(2)]),
         ("Hadamard", [1/math.sqrt(2), -1/math.sqrt(2)], [0, 1]),
-    ])
+    ]
+
+    test_data_no_parameters_inverses = [(item[0] + Operation.string_for_inverse, item[1], item[2])
```

Comment:
I would actually just duplicate the list and write it out with `"S.inv"`, etc., or maybe have a small function
```python
def inv(name):
    return name + Operation.string_for_inverse
```
for better readability, i.e. `inv("S")`.

Doing it programmatically is usually just confusing.

Reply:
Changed this for a number of particular cases for better readability. Thanks!